### PR TITLE
Enable/disable color picker based on useCustomColors on web theme

### PR DESCRIPTION
**Pull request by thescubageek**
Disable color picker on Web Themes if useCustomColors is set to false

### CODE REVIEW

File: app/components/spectrum-color-picker.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
 import SpectrumColorPickerComponent from 'ember-spectrum-color-picker/components/spectrum-color-picker';
- 
+
 export default SpectrumColorPickerComponent.extend({
   showInput: true,
   preferredFormat: 'hex',
+  updateDisabled: Ember.observer('disabled', function () {
+    this.$().spectrum(this.get('disabled') ? 'disable' : 'enable');
+  }),
```
Comment (thescubageek) [inline app/components/spectrum-color-picker.js]: The ember-spectrum-color-picker docs (https://www.npmjs.com/package/ember-spectrum-color-picker) say that the `disabled` property supports two-way bindings, but they are full of :hankey: ..... had to add this Observer based on the original jQuery Spectrum plugin to make it work

**Comment by thescubageek** [comment]
@jcstringer @mmitchellg5 This disables the color pickers when useCustomColors is false...

**Comment by thescubageek** [comment]
I also made a PR against the original ember component (https://github.com/RSSchermer/ember-spectrum-color-picker/pull/4) so hopefully this will part of the actual ember component soon, making our extension of `spectrum-color-picker.js` redundant.

**Comment by thescubageek** [comment]
My PR against the original ember component was accepted and merged into master (https://github.com/RSSchermer/ember-spectrum-color-picker/pull/4), so I've removed my disabled binding from https://github.com/G5/orion-cms/commit/57e5baa77b51ce06e7e5a06f478e26d6ca493a2e and updated the version of ember-spectrum-color-picker instead

**Comment by jcstringer** [comment]
Nice work dude, kudos for racking up a good open source contribution as part of this fix. :ship:

**Pull request merged**